### PR TITLE
Enter key must be pressed to proceed from info screens

### PR DESCRIPTION
**Pull request by Sindex42**

### CODE REVIEW

**Review by hotblack86 — APPROVED** [review]
Great. This should improve the user experience.

**Review by CKKH — APPROVED** [review]
Perfect for button bashers like me :joystick:

File: src/game.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def wait_for_key(self):
                     waiting = False
                     self.playing = False
                     pg.quit()
-                if event.type == pg.KEYDOWN:
+                if pg.key.get_pressed()[pg.K_RETURN]:
```
Comment (michaelnguyen974) [inline src/game.py:183]: Nice, now our users will not accidentally go to the next level

**Review by michaelnguyen974 — APPROVED** [review]

**Review by matharotheelf — APPROVED** [review]
Great addition!

**Pull request merged**